internal/engine/upper_test.go

```diff
@@ -2275,66 +2275,6 @@ func TestUpperPodRestartsBeforeTiltStart(t *testing.T) {
 	assert.NoError(t, err)
 }
 
-// This tests a bug that led to infinite redeploys:
-// 1. Crash rebuild
-// 2. Immediately do a container build, before we get the event with the new container ID in (1). This container build
-//    should *not* happen in the pre-(1) container ID. Whether it happens in the container from (1) or yields a fresh
-//    container build isn't too important
-func TestUpperBuildImmediatelyAfterCrashRebuild(t *testing.T) {
```

Comment:
i removed this test because I feel like the conditions it's testing (a pod crashes and tilt immediately starts building) shouldn't happen anymore. but open to suggestions on what might replace it.